### PR TITLE
Update French translation  #2396

### LINKED ISSUE #2396: [Bug] French translation needs to be completed
### Describe the issue

The diff output from the debug menu list 268 missing strings, so let's say there have been some lag^^

### Steps to Reproduce

1. Run the game in French

2. See a lot of English

### Expected Behaviour

French sentences

### Save Game

Save game attached to report: Yes/No

### CorsixTH Version

0.67

### Operating System

Linux Mint

### Theme Hospital Version

GOG.com

### Gamelog.txt

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by nMustaki**
*Fixes #2396 *

**Describe what the proposed change does**
- Add missing strings
- Remove unneeded ones
- Format the file

The game is starting and running in French, is there anything else to test ?

### CODE REVIEW

File: CorsixTH/Lua/languages/french.lua [hunk 1]
```diff
@@ -1189,4 +1367,201 @@ map_editor_window = {
...
+original_credits[352] = " "
+original_credits[353] = " "
+original_credits[354] = " "
+original_credits[355] = " "
+original_credits[356] = " "
+original_credits[357] = " "
+original_credits[358] = " "
+original_credits[359] = " "
+original_credits[360] = " "
+original_credits[361] = "."
```
Comment (lewri) [inline CorsixTH/Lua/languages/french.lua:1567]: Original credits are a bit weird.  I think each language was unique in the number of lines created.

**Review by lewri — COMMENTED** [review]
All good, though we don't need the string to be on a new line to the variable name (even though they go over 80-90chars). Not that it should really matter.

**Comment by nMustaki** [comment]
Yes the default style guide from vscode extension is a bit restrictive on line length, I'll look into that.
Do you want me to update the commit with longueur lines length before merging?

**Comment by lewri** [comment]
> Yes the default style guide from vscode extension is a bit restrictive on line length, I'll look into that. Do you want me to update the commit with longueur lines length before merging?

Preferably yes, but it's not a major thing imo.
You can likely fix the line break in notepad or on Github itself -- depending whether you wanted to force push or not.

**Comment by nMustaki** [comment]
Ok done. Is there a style guide I could convert into a .editorconfig ?

**Comment by lewri** [comment]
> Ok done. Is there a style guide I could convert into a .editorconfig ?

Yes here you go https://github.com/CorsixTH/CorsixTH/wiki/Coding-Conventions

File: CorsixTH/Lua/languages/french.lua [hunk 2]
```diff
@@ -16,19 +16,18 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE. --]]
-
-Language(utf8 "Français", "French" , "fr", "fre", "fra")
+SOFTWARE. --]] Language(utf8 "Français", "French", "fr", "fre", "fra")
```
Comment (lewri) [inline CorsixTH/Lua/languages/french.lua:19]:
```suggestion
SOFTWARE. --]]

Language(utf8 "Français", "French", "fr", "fre", "fra")
```
I'd keep the spacing here

File: CorsixTH/Lua/languages/french.lua [hunk 3]
```diff
@@ -995,48 +1085,39 @@ information = {
...
+          "Les internes et les médecins peuvent augmenter leurs compétences auprès d'un consultant dans la salle de formation. Si le consultant a des qualifications particulières (chirurgien, psychiatre ou chercheur), il transférera ses connaissances à ses élèves.", "Avez-vous essayé d'entrer le numéro d'urgence Européen (112) dans le fax ? Vérifiez que vous avez du son !",
+          "Vous pouvez ajuster certains paramètres tels que la résolution et la langue dans la fenêtre d'options accessible à la fois depuis le menu principal et pendant le jeu.", "Vous avez choisi une autre langue que l'anglais, mais il y du texte en anglais partout ? Aidez-nous à traduire les textes manquants dans votre langue !",
+          "L'équipe de CorsixTH cherche du renfort ! Vous êtes intéressé à coder, traduire ou faire des graphismes pour CorsixTH ? Contactez-nous sur notre Forum, Liste de Diffusion ou Canal IRC (#corsix-th sur freenode).", "Si vous avez trouvé un bug, SVP, reportez-le sur notre gestionnaire de bugs : th-issues.corsix.org.",
```
Comment (lewri) [inline CorsixTH/Lua/languages/french.lua:1102]: This one needs adjustment if possible to the new string:
https://github.com/CorsixTH/CorsixTH/pull/2259

File: CorsixTH/Lua/languages/french.lua [hunk 4]
```diff
@@ -995,48 +1087,39 @@ information = {
...
```
Comment (nMustaki) [inline CorsixTH/Lua/languages/french.lua:1104]:
```suggestion
          "L'équipe de CorsixTH cherche du renfort ! Vous êtes intéressé à coder, traduire ou faire des graphismes pour CorsixTH ? Contactez-nous sur notre serveur Discord, Sub-Reddit or serveur Matrix. Les liens sont sur notre site internet (CorsixTH.com).",
```

**Review by lewri — APPROVED** [review]

**Pull request merged**